### ISSUE TITLE
Publish to new album does not work

### ISSUE THREAD

**Issue by baghuis** [stanaka/lightroom-google-photo-plugin#1]
Hi Stanaka, thanks for this plugin. I can sync photos to Google Photos but I cannot create a new album and publish my photos. When I press publish I get an error pop-up: Can't update this collection. An internal error has occurred. Invalid at the top level of the document. Any ideas?
![image](https://cloud.githubusercontent.com/assets/25726495/22864822/a5c40612-f158-11e6-8c79-d599ef7ea069.png)

**Comment by stanaka** [comment]
Thank you for using it. I'll investigate it.

This plugin outputs a log at `~/Documents/GPhotoAPI.log`. Could you send me (shinji.tanaka@gmail.com) this log? It's very helpful for investigating the behaviour.

**Comment by swensorm** [comment]
In case it helps; I was getting this same error.  The log message stated `Insert is no longer supported`.

I haven't been able to get the plugin to create a new album, however if I create the album on the web and then also create it in lightroom (using the exact same name) then everything works.  Little annoying since the web interface doesn't let you create an empty album.

We'll see how long it keeps working given how much google is trying to choke the picasa api to death. 🤞

**Comment by SH4DY** [comment]
Yep same error here but the workaround works. Creating a new album with the same name on the GPhotos UI and then publishing. Need to get into Lua haha

**Comment by flipswitchingmonkey** [comment]
The problem is that Google has castrated the Picasaweb API for some strange reason: https://developers.google.com/picasa-web/docs/2.0/developers_guide_protocol#AddAlbums

This means that while you can still _post_ into an album, you can't create one any more via the API.

**Comment by stanaka** [comment]
I'm going to migrate new Google Photos API. Please wait for a while.
https://github.com/stanaka/lightroom-google-photo-plugin/pull/15
